test/mitmproxy/tools/web/test_app.py

```diff
@@ -275,3 +276,13 @@ def test_websocket(self):
         # trigger on_close by opening a second connection.
         ws_client2 = yield websocket.websocket_connect(ws_url)
         ws_client2.close()
+
+    def test_generate_tflow_js(self):
+        tflow_json = _json.dumps(
+            app.flow_to_json(tflow.tflow(resp=True, err=True)), indent=4, sort_keys=True
+        )
+        web_root = os.path.join(os.getcwd(), 'web')
```

Comment:
`cwd` is a bit dangerous as this may change depending on where you execute your tests. Use [this](https://github.com/mitmproxy/mitmproxy/blob/af0ad494348d0f2d773232b5913d5c86787e39f5/setup.py#L10) and construct the path relative from here. :)